app/assets/stylesheets/form.scss

```diff
@@ -0,0 +1,30 @@
+$font-family-sans-serif: "Ek Mukta", "Helvetica Neue", Helvetica, Roboto, Arial, sans-serif;
+
+.date.required.form-control{
+display: inline-block;
+width: 32%;
```

Comment:
Line should be indented 2 spaces, but was indented 0 spaces<br>Properties should be ordered display, margin-right, width